static/js/containers/LearnerSearchPage.js

```diff
@@ -41,21 +49,32 @@ class LearnerSearchPage extends React.Component {
   };
 
   render () {
-    const { currentProgramEnrollment, openEmailComposer } = this.props;
+    const {
+      currentProgramEnrollment,
+      openEmailComposer,
+      programs,
+      profiles
+    } = this.props;
+    const loaded = R.none(isProcessing, [
+      R.pathOr(null, [SETTINGS.user.username, 'getStatus'], profiles),
+      R.pathOr(null, ['getStatus'], programs)
+    ]);
```

Comment:
why are we checking for `profiles` and `programs` here? it seems like that's probably not the data we're waiting on here - maybe instead we should show the spinner until we first get a search result, for example.